scripts/change-log-generator.js

```diff
@@ -54,19 +54,51 @@ const typesToIgnore = [
   'revert'
 ];
 
+const RELEASE_OVERRIDE_ARG = '-o';
+const RELEASE_DATE_ARG = '-t';
+const VERBOSE_LOGGING_ARG = '-v';
+const PACKAGES_TO_IGNORE_ARG = '-i';
+
+function getArgumentValue(arg) {
+  const argIndex = process.argv.indexOf(arg);
+  if (argIndex > -1) {
+    const argValue = process.argv[argIndex + 1];
+    return argValue && !argValue.startsWith('-') ? argValue : '';
+  } else {
+    return '';
+  }
+}
+
+function checkErrorCode(code, errorMessage) {
+  if (code !== 0) {
+    console.log(errorMessage);
+    process.exit(-1);
+  }
+}
+
+function updateBranches(baseBranch) {
```

Comment:
Porting this logic over from the port-PR script. This just ensures everything is up to date before the automation begins.